### PR TITLE
German Oil Temperatures

### PR DESCRIPTION
**Pull request by itsjustdel**

### CODE REVIEW

File: Assets/Scripts/DialsManager.cs [hunk 1]
```diff
@@ -246,12 +263,12 @@ private void AsignOilTempIn()
 
         if (airplaneData.planeType == "P-38J-25")
         {
-            /*
+
```
Comment (itsjustdel) [inline Assets/Scripts/DialsManager.cs:266]: Probably don't need this whitespace

File: Assets/Scripts/DialsManager.cs [hunk 2]
```diff
@@ -246,12 +263,12 @@ private void AsignOilTempIn()
...
         if (airplaneData.planeType == "P-38J-25")
         {
-            /*
+            
             GameObject needleLargeL = oilTempInObjects[0].transform.Find("Dial").Find("Needle Large L").gameObject;
             countryDialBoard.GetComponent<RotateNeedle>().oilTempInNeedles.Add(needleLargeL);
             GameObject needleLargeR = oilTempInObjects[0].transform.Find("Dial").Find("Needle Large R").gameObject;
             countryDialBoard.GetComponent<RotateNeedle>().oilTempInNeedles.Add(needleLargeR);
-            */
+
```
Comment (itsjustdel) [inline Assets/Scripts/DialsManager.cs:271]: And whitespace here too

File: Assets/Scripts/DialsManager.cs [hunk 3]
```diff
@@ -957,6 +1010,18 @@ void DeactivateUnavailableDials(GameObject countryDialBoard, PlaneDataFromName.P
...
+        List<GameObject> allOilComboTemps = new List<GameObject>();
+        allOilComboTemps.AddRange(allOilComboTempsArray);
+
+        foreach (GameObject oilTemp in oilTempComboObjects)
+            allOilComboTemps.Remove(oilTemp);
+
+        for (int i = 0; i < allOilComboTemps.Count; i++)
+        {
+            allOilComboTemps[i].SetActive(false);
+        }
```
Comment (itsjustdel) [inline Assets/Scripts/DialsManager.cs:1024]: If not using brackets for one line loops (as per line `1019`, keep it consistent

File: Assets/Scripts/PlaneDataFromName.cs [hunk 4]
```diff
@@ -974,6 +987,17 @@ public static PlaneAttributes AttributesFromName(string name)
                 planeAttributes.waterTempType = DialVariant.A;
                 break;
 
+            // Couldn't find!
```
Comment (itsjustdel) [inline Assets/Scripts/PlaneDataFromName.cs:990]: Usually a reference to where it is in the source is here, this time I couldn't find it

File: Assets/Scripts/UDPClient.cs [hunk 5]
```diff
@@ -265,10 +265,34 @@ void ProcessPackage(byte[] bytes)
 			airplaneData.verticalSpeed = floats[6];
 			airplaneData.turnCoordinatorBall = floats[7];
 			airplaneData.turnCoordinatorNeedle = floats[8];
+
+			if(airplaneData.planeType == "Ju-52/3m g4e")
+            {
```
Comment (itsjustdel) [inline Assets/Scripts/UDPClient.cs:270]: Formatting needs fixed

File: Assets/Scripts/UDPClient.cs [hunk 6]
```diff
@@ -265,10 +265,34 @@ void ProcessPackage(byte[] bytes)
...
+				floats[27] = floats[28];
+				floats[28] = temp;
+
+			}
+
 			airplaneData.rpms[0] = floats[9];
 			airplaneData.rpms[1] = floats[10];
 			airplaneData.rpms[2] = floats[11];
 			airplaneData.rpms[3] = floats[12]; //support for 4 engines (you never know!)
+
```
Comment (itsjustdel) [inline Assets/Scripts/UDPClient.cs:294]: Extra whitespace needed?

Comment (itsjustdel) [reply Assets/Scripts/UDPClient.cs:294]: Why yes, it helps cut up the long list, but perhaps just one line is enough

**Comment by erikesund** [comment]
Great work

**Pull request merged**